### PR TITLE
feat: add tooltips to sidebar tabs

### LINKED ISSUE #2429: [Task]: Adjust left navigation
### Task description

TBD

### Requirements

TBD

### Acceptance criteria

TBD

### Creation checklist

- [X] I have assigned this task to the correct people
- [X] I have added the most appropriate labels
- [X] I have linked the correct milestone and/or project

### PR DESCRIPTION
**Pull request by begonaalvarezd**
## Summary

This PR adjusts the sidebar to have tooltips

Note ⚠️  this branch derives from the one in #2427, so I recommend merging that PR before.

### Changelog
```
feat: add sidebar tab component with tooltips
```

## Relevant Issues

Closes #2429

## Type of Change
Please select any type below that is applicable to your changes, and delete those that are not.
- [ ] __Breaking__ - any change that would cause existing functionality to not work as expected
- [ ] __Chore__ - refactoring, build scripts or anything else that isn't user-facing
- [ ] __Docs__ - changes to the documentation
- [ ] __Fix__ - a change which fixes an issue
- [x] __New__ - a change which implements a new feature
- [x] __Update__ - a change which updates existing functionality

### Platforms
Please select any platforms where your changes have been tested.
- __Desktop__
	- [ ] MacOS
	- [x] Linux
	- [ ] Windows
- __Mobile__
	- [ ] iOS
	- [ ] Android

### Instructions
Please describe the specific instructions, configurations, and/or test cases necessary to __test and verify__ that your changes work as intended.

## Checklist
Please tick all of the following boxes that are relevant to your changes, and delete those that are not.
- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or modified tests that prove my changes work as intended
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have verified that my latest changes pass CI workflows for testing and linting
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/shared/components/Sidebar.svelte [hunk 1]
```diff
@@ -19,13 +19,47 @@
...
 
+    $: $dashboardRoute, $activeProfile?.hasVisitedStaking || showStakingNotification, updateSidebar()
+
+    let sidebarTabs = [
+        {
+            active: false,
+            icon: 'wallet',
+            label: locale('tabs.wallet'),
+            route: Tabs.Wallet,
+            notificationType: undefined,
```
Comment (jeeanribeiro) [inline packages/shared/components/Sidebar.svelte:36]: I'm just thinking if we could have something like a Enum to list the different possible values of notificationType

File: packages/shared/components/Sidebar.svelte [hunk 2]
```diff
@@ -19,13 +19,47 @@
...
+            onClick: openStaking,
+        },
+    ]
+
+    function updateSidebar() {
+        sidebarTabs = sidebarTabs.map((tab) => {
+            tab.active = tab.route === $dashboardRoute
+            if (Tabs.Staking === tab.route) {
+                tab.notificationType = !$activeProfile?.hasVisitedStaking
+                    ? 'warning'
```
Comment (jeeanribeiro) [inline packages/shared/components/Sidebar.svelte:54]: I think we should use the NotificationType Enums here

**Review by jeeanribeiro — CHANGES_REQUESTED** [review]

File: packages/shared/components/Sidebar.svelte [hunk 3]
```diff
@@ -19,13 +19,47 @@
...
-    const hasTitleBar = document.body.classList.contains('platform-win32')
     const profileColor = 'blue' // TODO: each profile has a different color
 
     $: profileInitial = getInitials($activeProfile?.name, 1)
     $: healthStatus = $networkStatus.health ?? 0
     $: $dashboardRoute, $stakingEventState, $partiallyUnstakedAmount, manageUnstakedAmountNotification()
 
+    $: $dashboardRoute, $activeProfile?.hasVisitedStaking || showStakingNotification, updateSidebar()
+
+    let sidebarTabs = [
```
Comment (Tuditi) [inline packages/shared/components/Sidebar.svelte:30]: Maybe it makes sense to define a SideBarTab type here?

Comment (Tuditi) [reply packages/shared/components/Sidebar.svelte:30]: Then it also makes it convenient to only pass one prop instead of 5 to SideBarTab

**Review by Tuditi — APPROVED** [review]

File: packages/shared/components/Sidebar.svelte (hunk 2 above)
Comment (begonaalvarezd) [reply packages/shared/components/Sidebar.svelte:54]: using global notification type now

File: packages/shared/components/Sidebar.svelte (hunk 3 above)
Comment (begonaalvarezd) [reply packages/shared/components/Sidebar.svelte:30]: updated

**Pull request merged**